tests/conftest.py

```diff
@@ -142,6 +142,11 @@ def aws_resources(aws_credentials):
                     "Action": "logs:CreateLogGroup",
                     "Resource": "*",
                 },
+                {
+                    "Action": "s3:GetObject",
+                    "Effect": "Allow",
```

Comment:
Move `Effect` before `Action`, so it matches the order of the fields in the previous statement.